trie/constants.py

```diff
@@ -14,3 +14,16 @@
 NODE_TYPE_LEAF = 1
 NODE_TYPE_EXTENSION = 2
 NODE_TYPE_BRANCH = 3
+
+# Constants for Binary Trie
+TWO_BITS = [bytes([0, 0]), bytes([0, 1]),
+            bytes([1, 0]), bytes([1, 1])]
+PREFIX_00 = TWO_BITS[0]
```

Comment:
*minor nitpick*

I would *suggest* not making this constant reference the previous one and just declaring it independently in the spirit of decoupling